phase1-coordinator/src/testing/resources/round.json

```diff
@@ -1,7 +1,7 @@
 {
     "version": 0,
     "height": 0,
-    "startedAt": "1970-01-01T00:01:01Z",
+    "startedAt": 61,
```

Comment:
Case in point ☝️ 
@julesdesmit this sounds familiar to what you were saying the issue was?

Reply:
Yep, that caused the disconnect with the state monitor! I guess we are using a different type somewhere.